### PR TITLE
docs: add french README

### PR DESCRIPTION
**Pull request by eze-kiel**
#### What is it?
- [ ] Bugfix (user facing)
- [x] Feature (user facing) -> doc improvement
- [ ] Codebase improvement (dev facing)
- [ ] Meta improvement to the project (dev facing)

#### Description of the changes in your PR

This PR adds the full french README (`doc/README.fr.md`, and adds its link in `README.md` at the root of the repo.

#### Before/After Screenshots/Screen Record

- Before:
- After:
<details>
  <summary>See screenshot</summary>

image

</details>

#### Fixes the following issue(s)

No code fixes.

#### APK testing

No need as the code is not impacted by this PR.

#### Due diligence
- [x] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

File: doc/README.fr.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+1. Sauvegardez vos données via Settings > Content > Export Database pour conserver votre historique, vos abonnements et vos listes de lecture.
+2. Désinstallez NewPipe
+3. Téléchargez l'APK depuis la nouvelle source et installez le.
+4. Restaurez les données de l'étape 1 via Settings > Content > Import Database
+
+<b>Note: lorsque vous importez une base de données dans l'application officielle, assurez vous de toujours vérifier que c'est celle que vous avez exportée _depuis_ l'application officielle. Si vous importez une base de données provenant d'un autre APK que l'officiel, il se peut que des choses cassent. Une telle action n'est pas supportée, et vous devriez le faire uniquement lorsque vous êtes absolument certains de savoir ce que vous faites</b>
+
+## Contribuer
+
+Que vous ayez des idées, des traductions, des changements de design, du nettoyage ed code, ou encore un changement de code majeur, toute aide ets la bienvenue. L'app s'améliore un peu plus à chaque contribution, peu importe qu'elle soit grosse ou petite ! Si vous aimeriez être impliqué, jetez un coup d'oeil à nos [notes pour contribuer](.github/CONTRIBUTING.md).
```
Comment (BrightDV) [inline doc/README.fr.md:112]:
```suggestion
Que vous ayez des idées, des traductions, des changements de design, du nettoyage de code, ou encore un changement de code majeur, toute aide est la bienvenue. L'app s'améliore un peu plus à chaque contribution, peu importe qu'elle soit grosse ou petite ! Si vous aimeriez être impliqué, jetez un coup d'oeil à nos [notes pour contribuer](.github/CONTRIBUTING.md).
```

File: doc/README.fr.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+Nous recommandons la méthode 1 pour la plupart des utilisateurs. Les APKs installés en suivant les méthodes 1 ou 2 sont compatibles les uns avec les autres (ce qui signifie que si vous installez NewPipe en utilisant une de ces deux méthodes, vous pouvez également mettre à jour NewPipe en utilisant l'autre), mais pas avec ceux installés en utilisant la méthode 3. Cela est dû au fait que la même clé de signature (la notre) est utilisée pour les méthodes 1 et 2, mais une différente clé (celle de F-Droid) est utilisée pour la 3. Construire un APK de débug en utilisant la méthode 4 ne requiert aucune clé. Les clés de signature aident à assurer que l'utilisateur ne s'est pas fait avoir et installe une mise à jour frauduleuse de l'app. Lorsque la méthode 5 est utilisée, chaque APK est signé avec une clé unique et aléatoire générée par les GitHub Actio,s; vous ne pouvez donc pas les mettres à jour. Vous devrez alors sauvegarder et restaurer les données de l'application à chaque fois que vous voudrez utiliser un nouvel APK.
```
Comment (BrightDV) [inline doc/README.fr.md:100]:
```suggestion
Nous recommandons la méthode 1 pour la plupart des utilisateurs. Les APKs installés en suivant les méthodes 1 ou 2 sont compatibles les uns avec les autres (ce qui signifie que si vous installez NewPipe en utilisant une de ces deux méthodes, vous pouvez également mettre à jour NewPipe en utilisant l'autre), mais pas avec ceux installés en utilisant la méthode 3. Cela est dû au fait que la même clé de signature (la notre) est utilisée pour les méthodes 1 et 2, mais une différente clé (celle de F-Droid) est utilisée pour la 3. Construire un APK de débug en utilisant la méthode 4 ne requiert aucune clé. Les clés de signature aident à assurer que l'utilisateur ne s'est pas fait avoir et installe une mise à jour frauduleuse de l'app. Lorsque la méthode 5 est utilisée, chaque APK est signé avec une clé unique et aléatoire générée par les GitHub Actions; vous ne pouvez donc pas les mettres à jour. Vous devrez alors sauvegarder et restaurer les données de l'application à chaque fois que vous voudrez utiliser un nouvel APK.
```

**Review by BrightDV — COMMENTED** [review]
I just found some typos, otherwise looks great!

File: doc/README.fr.md (hunk 2 above)
Comment (BrightDV) [reply doc/README.fr.md:100]: `GitHub Actions` instead of `GitHub Actio,s`

**Comment by eze-kiel** [comment]
Nice catches! I'm fixing them right now

**Review by BrightDV — COMMENTED** [review]
I have switched to my computer and found some other small typos.
Also, you should add the link to french READMe inside the others READMEs (not only in the english one).
After that, it looks go to me!

**Comment by eze-kiel** [comment]
hi @BrightDV, thanks for your review!
it should be good this time :)

**Review by BrightDV — APPROVED** [review]
Yes !

**Comment by BrightDV** [comment]
Can someone merge this, please?

**Review by Green-linux — APPROVED** [review]

**Comment by TobiGr** [comment]
Thank you for your work. Your changes were merged in #10240

**Pull request closed without merging**